src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/signin/AttachedSignInPage.java

```diff
@@ -0,0 +1,96 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+import com.wikia.webdriver.elements.mercury.pages.ErrorPage;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.elements.mercury.pages.ErrorPage'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)